chromium_src/chrome/browser/ui/webui/favicon_source.cc

```diff
@@ -6,13 +6,59 @@
 #include "build/build_config.h"
 
 #if defined(OS_ANDROID)
+#include "extensions/buildflags/buildflags.h"
```

Comment:
buildflags don't have to be guarded by anything else

Reply:
cc: @samartnik

Reply:
Moved out of guard https://github.com/brave/brave-core/commit/7dbee7902a5aa183a171383a129ea45efdb7d837